sources/packages/web/src/types/contracts/GeneralStatusForBadge.ts

```diff
@@ -3,6 +3,9 @@ export enum GeneralStatusForBadge {
   InActive = "inactive",
   ActiveRestriction = "active restriction",
   ResolvedRestriction = "resolved",
+  Approved = "approved",
```

Comment:
Are we putting every single status for a badge in one single Enum?

Reply:
updated